src/frontend/src/pages/PIAIntakeForm/messages.ts

```diff
@@ -124,10 +124,10 @@ export default {
         en: `Cancel`,
       },
       TitleText: {
-        en: `Submit for MPO review?`,
+        en: `Submit for PIA Intake?`,
       },
       ParagraphText: {
-        en: `Your PIA status will be changed to MPO review. Your MPO will be notified that your edits are complete and your PIA is ready for review.`,
+        en: `Are you sure you want to submit your PIA intake?`,
```

Comment:
Same here. Is this change in text mentioned in the ticket?

Reply:
Not in the ticket itself, but in the figma design mock up.